mtags-interfaces/src/main/java/scala/meta/pc/ReferenceCountProvider.java

```diff
@@ -0,0 +1,7 @@
+package scala.meta.pc;
+
+
+public interface ReferenceCountProvider {
+
+    Integer references(String buildTargetIdentifier, String symbol);
```

Comment:
I wonder whether we should make it more generic. Like `workspacePriority` which we could modify on Metals side.